llvm/test/CodeGen/PowerPC/absdiff.ll

```diff
@@ -0,0 +1,19 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 4
+; RUN: opt -mtriple=powerpc64-linux-gnu -passes=instcombine < %s -o - | llc -mtriple=powerpc64-linux-gnu -o - | FileCheck %s
+
+define  i64 @absdiff(i64 %0, i64 %1) {
```

Comment:
Will the combination differ by triple? If this is `opt` change, I don't think we should mix `opt` runlines in codegen test to 'prove' combine result is better.